### PR TITLE
Ensure synchronization after flush

### PR DESCRIPTION
**Pull request by liu15**

### CODE REVIEW

File: src/care/LoopFuser.h [hunk 1]
```diff
@@ -1257,15 +1258,18 @@ void LoopFuser<REGISTER_COUNT, XARGS...>::registerAction(const char * fileName,
...
 
+// Flush the fusible actions if needed (if we have reached the buffer limit, for example).
+// All fusers are flushed asynchronously except the active observer, which is flushed synchronously
+// to prevent possible corruption of the previous loop with buffer information for the next loop.
 #define FUSIBLE_FLUSH_IF_NEEDED  if (FusedActions::flush_now) { \
                                     for ( FusedActions *__fuser__ : { \
                                        static_cast<FusedActions *> (LOOPFUSER(256)::getInstance()),\
                                        static_cast<FusedActions *> (LOOPFUSER(128)::getInstance()),\
-                                       static_cast<FusedActions *> (LOOPFUSER(64)::getInstance()),\
-                                       FUSED_ACTION_INSTANCE_32 FUSED_INSTANCE_COMMA \
```
Comment (adayton1) [inline src/care/LoopFuser.h:1265]: Did you mean to remove FUSED_ACTION_INSTANCE_32?

Comment (liu15) [reply src/care/LoopFuser.h:1265]: I did not.  Good catch!

**Review by adayton1 — APPROVED** [review]
LGTM

**Review by robinson96 — APPROVED** [review]
LGTM

**Pull request merged**